Specify rules for TLV duplicates (#8)
MikeBishop/quic-samsara: draft-yuan-quic-congestion-data.md

understand, but MAY decline to return protected statistics to a peer if it
cannot verify that it is willing to share the contained information.

> A receiver may get a mesage with multiple occurences of a particular TLV

Reviewer: ```suggestion
A receiver may get a message with multiple occurrences of a particular TLV
```

value. If the values are identical, the reciever SHOULD ignore them.

> If they differ, and one of the values is protected by an integrity,

Reviewer: ```suggestion
If they differ, and one of the values is protected by an integrity tag,
```

> the receiver MAY treat this as an attack and close the connection.

Reviewer: ```suggestion
the receiver SHOULD treat this as an attack and close the connection.
```
I think this path should be a little stronger, particularly as this behavior is already an option below.

> If none of the instances are integrity-protected, the receiver may

Reviewer: ```suggestion
If none of the instances are integrity-protected, the receiver MAY
```
